compiler/rustc_target/src/spec/aarch64_unknown_trusty.rs

```diff
@@ -0,0 +1,27 @@
+// Trusty OS target for AArch64.
+
+use super::{PanicStrategy, RelroLevel, Target, TargetOptions};
+
+pub fn target() -> Target {
+    Target {
+        llvm_target: "aarch64-unknown-linux-musl".into(),
+        pointer_width: 64,
+        data_layout: "e-m:e-i8:8:32-i16:16:32-i64:64-i128:128-n32:64-S128".into(),
+        arch: "aarch64".into(),
+        options: TargetOptions {
+            features: "+neon,+fp-armv8".into(),
+            executables: true,
+            max_atomic_width: Some(128),
+            panic_strategy: PanicStrategy::Abort,
+            os: "trusty".into(),
+            position_independent_executables: true,
+            crt_static_default: true,
+            crt_static_respected: true,
```

Comment:
I think this needs to be false. Otherwise rustc will respect `-Ctarget-features=-crt-static` and force dynamic linking of libc in that case.

Reply:
You're right, I've updated it accordingly.